ChartsDemo-macOS/PlaygroundChart.playground/Pages/LineChart.xcplaygroundpage/Contents.swift

```diff
@@ -59,8 +59,8 @@ rightAxis.drawGridLinesEnabled = false
 rightAxis.granularityEnabled = false
 //: ### Legend
 let legend = chartView.legend
-legend.font = NSUIFont(name: "HelveticaNeue-Light", size: CGFloat(12.0))!
-legend.textColor = NSUIColor.black
+legend.font = NSFont(name: "HelveticaNeue-Light", size: CGFloat(12.0))!
```

Comment:
`.init(name:size)`
Removing the platform safe implementation makes this less portable. If you want to hide it (which I'm inclined to agree with), take advantage of the type inference.

Reply:
Fixed
Thanks 👍

Reply:
@jjatie `.init(name: size:)` seems that do not work

```
error: LineChart.xcplaygroundpage:56:61: error: type of expression is ambiguous without more context
legend.font = .init(name: "HelveticaNeue-Light", size: 12.0)!
```